### PR TITLE
WIP tag name --> tag key, field name --> field key

### LINKED ISSUE #2701: update "tag name" to "tag key" and "field name" to "field key" for consistency and clarity
I would prefer to standardize on "key" and "value" for tags and fields rather than "name" and "value", since they are key-value pairs. Many error messages still refer to tag name or field name.

### PR DESCRIPTION
**Pull request by beckettsean**
fixes #2701

### CODE REVIEW

**Comment by otoolep** [comment]
+1 on green.

**Comment by beckettsean** [comment]
@gunnaraasen can you check my work on this?

File: QUERIES.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-The top level name is called a measurement. These names can contain any characters. Then there are field names, field values, tag keys and tag values, which can also contain any characters. However, if the measurement, field, or tag contains any character other than [A-Z,a-z,0-9,_], or if it starts with a digit, it must be double-quoted. Therefore anywhere a measurement name, field name, field value, tag name, or tag value appears it should be wrapped in double quotes.
+The top level name is called a measurement. These names can contain any characters. Then there are field names, field values, tag keys and tag values, which can also contain any characters. However, if the measurement, field, or tag contains any character other than [A-Z,a-z,0-9,_], or if it starts with a digit, it must be double-quoted. Therefore anywhere a measurement name, field key, field value, tag key, or tag value appears it should be wrapped in double quotes.
```
Comment (gunnaraasen) [inline QUERIES.md]: > Therefore anywhere a measurement name, field key, field value, tag key, or tag value appears it should be wrapped in double quotes.

This doesn't sound right. Tag values, field values, and passwords are interpreted as string literals and should be single quoted. Measurement names, tag keys, field keys, and any other names are identifiers. Identifiers are double quoted unless they fit certain conditions.

**Comment by gunnaraasen** [comment]
One comment on something that was already wrong before this PR, otherwise +1.

**Comment by beckettsean** [comment]
Addressed @gunnaraasen's concern. Will merge on green tests

**Pull request merged**